web/frontend/angular/src/app/main-pages/register/register.component.ts

```diff
@@ -36,7 +38,20 @@ export class RegisterComponent implements OnInit {
         if (this.authForm.invalid) {
             return
         }
-        this.authService.register(this.authForm.value)
-        this.router.navigateByUrl("homepage")
+        this.authService.register(this.authForm.value).subscribe(
+            (res: any) => {
+                console.log(res)
+                this.router.navigateByUrl("homepage")
+            },
+            (error) => {
+                console.log(error)
+                $("input[type=email],input[type=password]").addClass("error")
+                $("#error").removeClass("d-none")
+                setTimeout(() => {
+                    $("input").removeClass("error")
+                    $("#error").addClass("d-none")
+                }, 2000)
```

Comment:
It's better to leave the error visible indefinitely to ensure the user does not miss it. The error should go away if the user successfully registers.

Reply:
@alinaah You did not address this comment.